idx/open-in-idx-template/README.md

```diff
@@ -0,0 +1,28 @@
+## Open in Project IDX 
+
+## How to use the template
+
+This template allows opening Google Maps Platform Android tutorial code samples in IDX IDE. To open a code sample in IDX, go to `https://idx.google.com/new?template=https://github.com/googlemaps-samples/android-samples/idx/open-in-idx-template` and provide the gitUrl, subdir and the name of the activity to launch as query params. 
+
+```
+https://idx.google.com/new
+?template=https://github.com/googlemaps-samples/android-samples/idx/open-in-idx-template 
+&giturl=https://github.com/googlemaps-samples/android-samples
+&subdir=tutorials/java/CurrentPlaceDetailsOnMap/
+&launchactivity=com.example.currentplacedetailsonmap/.MapsActivityCurrentPlace
+&apikey=AIXXXXXXXXXXXXXXsg
```

Comment:
the "sg" should not be included, just "AIza" is the standard prefix

Reply:
Done.